leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/tlv/LwM2mNodeTlvDecoder.java

```diff
@@ -49,6 +52,22 @@ public class LwM2mNodeTlvDecoder implements NodeDecoder {
 
     private static final Logger LOG = LoggerFactory.getLogger(LwM2mNodeTlvDecoder.class);
 
+    // parser used for core link parser
+    private final LinkParser linkParser;
+
+    public LwM2mNodeTlvDecoder() {
+        this(new DefaultLwM2mLinkParser());
+    }
+
+    /**
+     * Create a new LwM2mNodeTlvDecoder with a custom link parser.
+     * 
+     * @param linkParser the link parser for core link format resources.
+     */
```

Comment:
cosmetic remark but you can use link when you refer class : 
E.g. `Create a new {@link LwM2mNodeTlvDecoder} with a custom {@link LinkParser}.`

Reply:
done